gradle/groovyProject.gradle

```diff
@@ -106,7 +106,10 @@ class ClasspathManifest extends DefaultTask {
 
     @TaskAction
     def generate() {
-        manifestFile.withOutputStream { properties.save(it, 'module definition') }
+        // We write this out ourself instead of using the properties class to avoid the
+        // annoying timestamp that insists on placing in there, that throws out incremental.
+        def content = getProperties().entrySet().collect { "$it.key=$it.value" }.sort().join("\n")
```

Comment:
This doesn't correctly encode and escape. We should at least comment here that we accept this, if we aren't going to fix it. The simplest way to fix is to use Properties, write to a String and chop off the first line.